app/Http/Controllers/DriverController.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Drivers;
+use Illuminate\Http\Request;
+
+class DriverController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $drivers = Drivers::withTrashed()->paginate(15);
```

Comment:
Kokia prasme istrinti irasus jei mes juos vistiek atvaizduojame?
Turetu buti du sarasai, viename turetu buti visi irasai, o kitame istrinti irasai. Destroy metodas turetu priimti tik not trashed irasus, o restore metodas priimti tik trashed irasus.